homeassistant/components/airtouch5/climate.py

```diff
@@ -152,6 +148,14 @@ def __init__(self, client: Airtouch5SimpleClient, ability: AcAbility) -> None:
         if ability.supports_mode_heat:
             self._attr_hvac_modes.append(HVACMode.HEAT)
 
+        self._attr_supported_features = (
+            ClimateEntityFeature.TARGET_TEMPERATURE
+            | ClimateEntityFeature.FAN_MODE
+            | ClimateEntityFeature.TURN_OFF
```

Comment:
We have an `async_turn_on`/`off` implementation in `Airtouch5Zone` below, but not in `Airtouch5AC` here.
Is there a default implementation we inherit, or do we need to add one?

Reply:
This adds `turn_off` by default as you support hvacmode off and then tests to add `turn_on` if any other hvacmode is added.

Below we add both directly as on/off methods are implemented.

Reply:
I see now there is a base implementation in ClimateEntity
https://github.com/home-assistant/core/blob/595dd651bb59ec78d96842c746e6340518ef41fd/homeassistant/components/climate/__init__.py#L742

Seems good 👍